source/wp-content/themes/wporg-developer/scss/prism.scss

```
@@ -209,6 +209,9 @@ pre.line-numbers > code {
 
 /* Copy button */
 .wp-code-block-button-container {
+	position: sticky;
+	top: var(--wp-global-header-height);
```

This `--wp-global-header-height` value doesn't include the admin bar height, so if you're logged in the button is slightly hidden. You can use the `--wp-admin--admin-bar--height` custom property to include it.

```suggestion
	top: calc(var(--wp-global-header-height) + var(--wp-admin--admin-bar--height));
```

---

Fixed in https://github.com/WordPress/wporg-developer/pull/125/commits/8c328d3dec4c625fdcea3ac34a0a0d7960cc360e